File: 2022_c#_basics/11-while-loop/03-sum-numbers.cs
using System;

namespace _03_Sum_Numbers
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int firstInput = int.Parse(Console.ReadLine());
            int sum = 0;

            while (firstInput > sum)
            {
                int secondInput = int.Parse(Console.ReadLine());
                sum += secondInput;
            }

            Console.WriteLine(sum);
        }
    }
}

File: 2024_java_basics/11-while-loops/tasks/03-sum-numbers.java
import java.util.Scanner;

public class SumNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int firstInput = scanner.nextInt();
        int sum = 0;

        while (firstInput > sum) {
            int secondInput = scanner.nextInt();
            sum += secondInput;
        }

        System.out.println(sum);

        scanner.close();
    }
}
